reduce-tta.html

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: "Time to allocate"
+bodyclass: ""
+---
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <h1 class="heading-xlarge">
+      <span class="heading-secondary">Guide</span>
+      How to reduce time to allocate
+    </h1>
+  </div>
+</div>
+<div class="grid-row">
+  <div class="column-two-thirds">
+      <h2 class="heading-medium">How to Improve</h2>
+      <p>Time to Allocate is an important thing to track and reduce for a few reasons:</p>
+      <ul class="list list-bullet">
+        <li>There’s <strong>less interaction with external systems</strong> so it's a fairly well defined problem</li>
+        <li><strong>Software can help</strong> by recording metrics, learning and suggesting actions</li>
+        <li><strong>Central Teams can make an impact</strong> as they have more control over this part of the system</li>
+      </ul>
+      <p>Once you can reliable monitor Time-to-Allocate, the first thing to do is set an SLA target.</p>
+      <p>Ideally, there shouldn’t be more than 48 hours for each handoff, making a total target time of 96 hours (4 working days) between the Citizen submitting the request and it being allocated to an Information Holder who can definitely answer it.</p>
+      <h2 class="heading-medium">Central Team to Champion Delays</h2>
+      <ul class="list list-bullet">
+        <li><a href="#">Too many Champions</a> – not obvious which to pick, so it takes several attempts (all taking time) before it arrives at the correct Champion.</li>
+        <li>Central Team don’t have a good enough understanding of information held by each Service Area. <a href="#">Build an information asset register</a> and provide it to Central Team. Keep it updated. Ideally <a href="#">build it in to the software</a>.</li>
+        <li>Champions aren’t trained enough to know that sitting on a request causes overall problems. No consequences for doing this. <a href="#">Improve training rollout</a> and <a href="#">consider introducing rewards and incentives for good performance</a>.
+          </li>
+      </ul>
+      <h2 class="heading-medium">Champion to Information Holder Delays</h2>
+      <ul class="list list-bullet">
+        <li>Not enough Champions – they’re responsible for too broad an area and don’t have enough knowledge of all the information they may have access to. Read ‘<a href="#">how many champions do we need?</a>’
+          </li>
+        <li>Information Holders aren’t trained enough to know that sitting on a request causes overall problems. No consequences for doing this. a href="#">Improve training rollout</a> and <a href="#">consider introducing rewards and incentives for good performance</a>.</li>
```

Comment:
First url is broken

Reply:
I've added `class="not-implemented"` to the fake links now